src/plugins/tabs/_noscript.scss

```diff
@@ -6,11 +6,12 @@
 /**
  * Mobile-friendly styles
  */
- 
+
 %tabs-noscript-details-summary-display-block-important {
 	> {
 		details {
 			display: block;
+			padding-left: $details-identation;
```

Comment:
It looks like the left padding is around 2x wider in small view now (compared to what's currently on github.io). In medium and bigger views the summaries look like they're going right up against the border.

Maybe there'd be value in removing that line and adding something like ``summary {margin-left: 0;}`` that only targets medium view and larger when JS is disabled.